js/src/common/components/Alert.tsx

```diff
@@ -1,31 +1,33 @@
-import Component from '../Component';
+import Component, { ComponentAttrs } from '../Component';
 import Button from './Button';
 import listItems from '../helpers/listItems';
 import extract from '../utils/extract';
+import Mithril from 'mithril';
```

Comment:
So do we need to import this or not? IIRC from the TypeScript discussions in the mithril-2 branch, I was told that we're exposing this namespace in the shims so that it does not need to be imported.

Reply:
Same in other file.

Reply:
I think this is the typing library, not the actual library. @datitisev knows more about this though

Reply:
But you are importing this only for the types, as far as I can tell?

Reply:
I don't entirely remember anything 👼 

Will have to check in IDE to see what it complains about.

Reply:
Should be tested out, I guess?

Reply:
@datitisev any luck?

Reply:
I have not tested code.

Reply:
After some discussion, we've concluded that importing this in core is not an issue, as mithril is used in other places anyway. The shim is used for recognition of the global `m`, but we still need to type stuff that's not part of that global function.

There might be issues with extensions and typescript, but that is unrelated to this PR, and should be revisited later.